README.md

```diff
@@ -1,2 +1,24 @@
-# pass-bakery
-scala training project
+# Pass-Bakery
+Scala training project
+
+## Project Dependencies
+* play json
+* jdbc (used by Scala in the Play Framework)
+* evolutions
+* postgres
+* doobie
+
+## Running the Application
+1. Install [Docker](https://www.docker.com/products/docker-desktop/) and  [postgresql](https://www.postgresql.org/download/) on your machine, if not installed already.
+2. Launch Docker (this app expects the latest postgres alpine)
+3. Run the database through terminal commands
```

Comment:
```suggestion
3. Run the database through terminal commands
    1. Run the database with the command:
    2. docker run --name pass-bakery-postgres -e POSTGRES_USER=user -e POSTGRES_PASSWORD=pass -p 5432:5432 -d postgres:alpine
```
You expect a specific user, database name and password as well as port. Do not make your prospective users or future developer guess how to do their setup.

You mention one time things below. Do these hold if someone stops the docker container? If not, they are not one time things, they need to be done everytime someone starts the database